src/Grid/grid.jl

```diff
@@ -435,6 +437,7 @@ function ExclusiveTopology(cells::Vector{C}) where C <: AbstractCell
     face_skeleton_local = Vector{FaceIndex}()
     fs_length = length(face_skeleton_global)
     for (cellid,cell) in enumerate(cells)
+        getdim(cell) == 1 && continue
```

Comment:
shouldnt the 1D case also have a "face" to be evaluated for e.g. DG formulations?

Reply:
Yes, but it is already stored in the vertex neighborhood there. I can mirror it, but integrating this weird corner case in the face construction loop is probably not worth it. What do you think?

Reply:
can we then just extra dispatch `face_skeleton`?
```julia
faceskeleton(top::ExclusiveTopology, grid::AbstractGrid{1}) =  vertex_neighborhood..
```

Reply:
hm but this doesn't work for embedded lines properly right?

Reply:
Maybe like this?

Reply:
But for mixed dimensional DG stuff you have different problems to solve :D